cfme/tests/intelligence/test_optimization.py

```diff
@@ -0,0 +1,68 @@
+import pytest
+from widgetastic.utils import attributize_string
+
+from cfme import test_requirements
+from cfme.utils.blockers import BZ
+
+pytestmark = [test_requirements.report, pytest.mark.tier(1), pytest.mark.ignore_stream("5.10")]
+
+REPORTS = [
+    "Host CPU Trends (last week)",
+    "Host Memory Trends (last week)",
+    "Offline VMs with Snapshot",
+    "Top CPU Consumers (weekly)",
+    "Top Memory Consumers (weekly)",
+    "VMs with Volume Free Space >= 75%",
+]
+
+
+@pytest.mark.meta(coverage=[1769346])
```

Comment:
Should be `automates`

Reply:
Um I thought of that, but the bug is only a small part of the test. I would usually use `automates` if the whole bug was automated by the test, not if the but was a small part of it. But let me know if you think otheriwse :)

Reply:
Maybe more discussion happened offline - I'd agree with @john-dupuy, automates does not imply that the entire (or even a majority) of the test content is concerned with the BZ. Merely that the BZ root cause is automated by this test case.